src/core_plugins/region_map/public/region_map_vis.js

```diff
@@ -31,7 +31,9 @@ VisTypesRegistryProvider.register(function RegionMapProvider(Private, regionmaps
         addTooltip: true,
         colorSchema: 'Yellow to Red',
         selectedLayer: selectedLayer,
-        selectedJoinField: selectedJoinField
+        selectedJoinField: selectedJoinField,
+        mapZoom: 2,
+        mapCenter: [0, 0]
```

Comment:
if you put it in uiState you'll need to init it when you create your chart there (if its not yet present)